cmd/agent/app/integrations.go

```diff
@@ -315,9 +316,19 @@ func installTuf(cmd *cobra.Command, args []string) error {
 	// Run pip check to determine if the installed integration is compatible with the base check version
 	pipErr := pipCheck(cachePath)
 	if pipErr == nil {
-		fmt.Println(color.GreenString(fmt.Sprintf(
-			"Successfully installed %s %s", integration, versionToInstall,
-		)))
+		err := moveConfigurationFiles(integration)
+		if err == nil {
+			fmt.Println(color.GreenString(fmt.Sprintf(
+				"Successfully installed %s %s", integration, versionToInstall,
+			)))
+		} else {
+			fmt.Println(color.GreenString(fmt.Sprintf(
+				"Installed %s %s", integration, versionToInstall,
+			)))
+			fmt.Println(color.RedString(fmt.Sprintf(
+				"Error while moving configuration files for %s: %v", integration, err,
+			)))
+		}
 		return nil
```

Comment:
I wasn't sure about that, but I thought it wasn't completely a failure since it most cases the integration should just keep working. Especially when the failure is about moving the example file. Might no be true for default configuration files though.
I will change and return an error though, makes more sense 👍